next/components/pages/RegulationDetailPageContent.tsx

```diff
@@ -0,0 +1,261 @@
+        <MLink
+          variant="underlined-medium"
+          className="whitespace-nowrap"
+          href={`/vzn/${cancellation?.attributes?.slug}`}
+        >
+          VZN {cancellation.attributes?.regNumber}
+        </MLink>{' '}
+        od dňa{' '}
+        <span className="font-semibold">{formatDate(cancellation?.attributes?.effectiveFrom)}</span>
+        .{' '}
+      </Typography>
+    ) : isAmendeningCancelled ? (
+      <Typography type="p" className="whitespace-normal">
+        Toto VZN je dodatkom{' '}
+        {cancelledAmendees.map((cancelledAmendee, index) => {
+          return (
+            <Fragment key={cancelledAmendee.id}>
+              {' '}
+              {index === 0 ? ' k ' : index === cancelledAmendees.length - 1 ? ' a k ' : ', k '}{' '}
+              <MLink
+                variant="underlined-medium"
+                className="whitespace-nowrap"
+                href={`/vzn/${cancelledAmendee.attributes?.slug}`}
+              >
+                {' '}
+                VZN {cancelledAmendee.attributes?.regNumber}
+              </MLink>
+              , ktoré bolo zrušené nariadením{' '}
+              <MLink
+                variant="underlined-medium"
+                className="whitespace-nowrap"
+                href={`/vzn/${cancelledAmendee.attributes?.cancellation?.data?.attributes?.slug}`}
+              >
+                {' '}
+                VZN {cancelledAmendee.attributes?.cancellation?.data?.attributes?.regNumber ?? ''}
+              </MLink>{' '}
+              s účinnosťou od{' '}
+              <span className="font-semibold">
+                {' '}
+                {formatDate(
+                  amending?.find((amended) => amended.attributes?.cancellation?.data)?.attributes
+                    ?.cancellation?.data?.attributes?.effectiveFrom,
+                )}
+              </span>
+            </Fragment>
+          )
+        })}
+        .{' '}
+      </Typography>
+    ) : (
+      <Typography type="p" className="whitespace-normal">
+        Toto VZN je aktuálne platné, s dátumom účinnosti od{' '}
+        <span className="font-semibold">{formatDate(regulation.attributes?.effectiveFrom)}</span>.{' '}
+      </Typography>
+    ),
+    button: isCancelled ? { title: 'Zobraziť zrušujúce VZN', handler: () => {} } : null,
+  }
+
+  return (
+    <div className="mb-8 flex flex-col gap-y-8">
+      {/* TODO use something else than alert */}{' '}
+      <div className="flex flex-col gap-y-2">
+        {' '}
+        <Typography type="h2" size="h4">
+          Dátum účinnosti{' '}
+        </Typography>
+        <Typography type="p">{formatDate(regulation.attributes?.effectiveFrom)}</Typography>{' '}
+      </div>{' '}
+      {isCancelled || isAmendeningCancelled ? (
+        <Alert
+          type={alertProps.type as any}
+          message={alertProps.message}
+          variant="message"
+          content={alertProps.content as any} // workaround for width, so it fits to implied grid
```

Comment:
Never use `as any` unless really necessary.
Eslint should not allow you to use it without error/warning.